src/constants/error/major/major-error-message.constant.ts

```diff
@@ -0,0 +1,7 @@
+import { ErrorMessage } from '@src/types/type';
+import { MAJOR_ERROR_CODE } from './major-error-code.constant';
+
+export const MAJOR_ERROR_MESSAGE: ErrorMessage<typeof MAJOR_ERROR_CODE> = {
+  [MAJOR_ERROR_CODE.ALREADY_EXIST_MAJOR_CODE]: 'A code that already exists.',
+  [MAJOR_ERROR_CODE.ALREADY_EXIST_MAJOR_NAME]: 'A name that already exists',
```

Comment:
```suggestion
  [MAJOR_ERROR_CODE.ALREADY_EXIST_MAJOR_CODE]: 'mMajor code that already exists.',
  [MAJOR_ERROR_CODE.ALREADY_EXIST_MAJOR_NAME]: 'A name that already exists',
```
처럼 정확히 어떤게 이미 존재하는지 알려주면 더 좋을거같습니다.